src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_InvalidExpression.vb

```diff
@@ -0,0 +1,328 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Semantics
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+
+        <Fact, WorkItem(17598, "https://github.com/dotnet/roslyn/issues/17598")>
+        Public Sub InvalidInvocationExpression_BadReceiver()
+            Dim source = <![CDATA[
+Imports System
+
+Class Program
+    Private Shared Sub Main(args As String())
+        Console.WriteLine2()'BIND:"Console.WriteLine2()"
+    End Sub
+End Class
+    ]]>.Value
+
+            ' This might change with https://github.com/dotnet/roslyn/issues/18069
+            Dim expectedOperationTree = <![CDATA[
+IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid)
+  IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid)
+    IOperation:  (OperationKind.None)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of InvocationExpressionSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact, WorkItem(17598, "https://github.com/dotnet/roslyn/issues/17598")>
+        Public Sub InvalidInvocationExpression_OverloadResolutionFailureBadArgument()
```

Comment:
Ah sorry, you meant VB. Yes it seems to be valid code. Let me tweak the test here for VB.